src/routes/routes.tsx

```diff
@@ -11,38 +11,42 @@ import Oppsummering from './klageskjema/oppsummering';
 import KvitteringPage from './klageskjema/kvittering/kvittering-page';
 import { INNGANG_KATEGORIER, Kategori } from '../kategorier/kategorier';
 import AppContextComponenet from '../app-context/app-context';
-import { MainContainer } from '../styled-components/common';
 import { Klage } from '../klage/klage';
 import { loggedInRedirect } from './loggedin-redirect';
 import NotFoundPage from './not-found-page';
 import RootWithQuery from './root-with-query';
 import InngangKategorier from './inngang/inngang-kategorier';
 import InngangInnsendingDigital from './inngang/inngang-innsendingsvalg-digital';
 import InngangInnsendingPost from './inngang/inngang-innsendingsvalg-post';
+import FormTitle from './form-title';
+import Layout from './layout';
 
 const App = () => (
     <React.StrictMode>
         <BrowserRouter>
             <AppContextComponenet>
-                <MainContainer>
-                    <ErrorBoundary>
-                        <Switch>
-                            {innsendingsRoutes}
-                            {kategoriRoutes}
-                            <Route path={'/ny'} exact>
-                                <UserLoader>
-                                    <CreateKlage />
-                                </UserLoader>
-                            </Route>
-                            <Route path={LOGGED_IN_PATH} render={loggedInRedirect} exact />
-                            <Route path={'/:klageId/begrunnelse'} render={begrunnelse} exact />
-                            <Route path={'/:klageId/oppsummering'} render={oppsummering} exact />
-                            <Route path={'/:klageId/kvittering'} render={kvittering} exact />
-                            <Route path={'/'} component={RootWithQuery} exact />
-                            <Route component={NotFoundPage} />
-                        </Switch>
-                    </ErrorBoundary>
-                </MainContainer>
+                <Layout>
```

Comment:
Om vi flytter `Layout` til sidenivåene og sender inn bakgrunnsfargen, så slipper vi å ha logikk i den.
Jeg har testet dette lokalt og kan pushe endringene.

Reply:
> Om vi flytter `Layout` til sidenivåene og sender inn bakgrunnsfargen, så slipper vi å ha logikk i den.
> Jeg har testet dette lokalt og kan pushe endringene.

Funker bra for meg :D nice!